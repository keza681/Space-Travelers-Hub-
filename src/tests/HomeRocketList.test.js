import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../Redux/ConfigureStore';
import RocketList from '../components/HomeRocket/HomeRocketList';

describe('List oof Rocket test ', () => {
  it('Should renders the Rocket page succefully', () => {
    const app = renderer.create(
      <Provider store={store}>
        <RocketList />
      </Provider>,
    ).toJSON();
    expect(app).toMatchSnapshot();
  });
});
