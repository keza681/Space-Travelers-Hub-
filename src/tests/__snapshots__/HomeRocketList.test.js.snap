// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List oof Rocket test  Should renders the Rocket page succefully 1`] = `
<div
  className="rockets"
/>
`;
